packages/react-components/src/components/SendBox.tsx

```diff
@@ -217,12 +242,36 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   };
 
   /* @conditional-compile-remove(file-sharing) */
-  const renderFileUploadErrorMessage: JSX.Element = useMemo(() => {
+  const uploadInProgressErrorBar = useMemo(() => {
+    const renderError = !!(
+      props.activeFileUploads?.length &&
+      !props.activeFileUploads
+        .filter((fileUpload) => !fileUpload.errorMessage)
+        .every((fileUpload) => fileUpload.isUploaded)
+    );
+    return renderError && <SendBoxErrorBar message={strings.fileUploadsPendingError} />;
```

Comment:
It looks like all the errorBars are sharing the same underlying component - SendBoxErrorBar, is it possible to only change error message string instead of rendering different component? (Could be easier if we separate logic out into a single error component like what I mentioned)

Reply:
I am afraid the logic is not as trivial. I think you understand that already. Let me know if you need me to explain more.